website/content/docs/operations/monitoring-nomad.mdx

```diff
@@ -111,28 +111,43 @@ The `nomad.raft.fsm.apply` metric is an indicator of the time it takes
 for a server to apply Raft entries to the internal state machine. If
 this number trends upwards, look at the `nomad.nomad.fsm.*` metrics to
 see if a specific Raft entry is increasing in latency. You can compare
-this to warn-level logs on the Nomad servers for "attempting to apply
-large raft entry". If a specific type of message appears here, there
+this to warn-level logs on the Nomad servers for `attempting to apply
+large raft entry`. If a specific type of message appears here, there
 may be a job with a large job specification or dispatch payload that
 is increasing the time it takes to apply raft messages.
 
-## Federated Deployments (Serf)
+## Scheduling
 
-Nomad uses the membership and failure detection capabilities of the Serf library
-to maintain a single, global gossip pool for all servers in a federated
-deployment. An uptick in `member.flap` and/or `msg.suspect` is a reliable indicator
-that membership is unstable.
+The [Scheduling] documentation describes the workflow of how evaluations become
+scheduled plans and placed allocations.
 
-If these metrics increase, look at CPU load on the servers and network
-latency and packet loss for the [Serf] address.
+### Progress
 
-## Scheduling
+There is a class of bug possible in Nomad where the two parts of the scheduling
+pipeline, the workers and the leader's plan applier, *disagree* about the
+validity of a plan. In the pathological case this can cause a job to never
+finish scheduling as workers produce the same plan the plan applier repeatedly
+rejects.
+
+While this class of bug if very rare, it can be detected by repeated log lines
+on the Nomad servers containing `plan for node rejcted`:
```

Comment:
```suggestion
While this class of bug is very rare, it can be detected by repeated log lines
on the Nomad servers containing `plan for node rejected`:
```